templates/legacy/templates.go

```diff
@@ -0,0 +1,314 @@
+// Package legacy contains variables holding embded legacy template files.
```

Comment:
```suggestion
// Package legacy contains variables holding embedded legacy template files.
```
Really irrelevant, but couldn't resist. :sweat_smile:

Reply:
Listo.